src/main/java/com/emc/ecs/sync/TargetFilter.java

```diff
@@ -80,14 +79,14 @@ public void filter(ObjectContext objectContext) {
             }
 
             // object needs to be updated
-            log.debug("updating object in target (source:{}, target:{})...",
+            log.info("updating object in target (source:{}, target:{})...",
```

Comment:
the only change to this class is to the log statement levels (debug -> info).. is that really necessary?